### PR TITLE
Release 2.9.6

### PR DESCRIPTION
**Pull request by YaphetKG**
- Adds Bacpac Parser
- Jenkins coverage and Build overhaul

### CODE REVIEW

**Review by waTeim — APPROVED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -1,5 +1,7 @@
 library 'pipeline-utils@master'
 
+CCV = ""
```
Comment (gaurav) [inline Jenkinsfile:3]: Could you please document this line so it's clearer what `CCV` is used for?

File: src/dug/core/parsers/bacpac_parser.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import logging
+from typing import List
+from xml.etree import ElementTree as ET
+
+from dug import utils as utils
+from ._base import DugElement, FileParser, Indexable, InputFile
+
+logger = logging.getLogger('dug')
```
Comment (gaurav) [inline src/dug/core/parsers/bacpac_parser.py:8]: Should this be a BACPACParser-specific parser so it's easier to figure out where the log messages are coming from? Or maybe it doesn't matter?

File: src/dug/core/parsers/bacpac_parser.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+
+class BACPACParser(FileParser):
+    # Class for parsers SPRINT Data dictionary into a set of Dug Elements
```
Comment (gaurav) [inline src/dug/core/parsers/bacpac_parser.py:12]:
```suggestion
    # Class for parsing BACPAC data dictionaries in dbGaP XML format into a set of Dug Elements.
```

**Review by gaurav — APPROVED** [review]
I haven't been able to get this working on my computer yet, but the code looks right to me! I've added a few minor suggestions and asked some questions, but these can be dealt with after this PR has been merged if there's a rush. I would explicitly document the new Git commit message check somewhere -- either in the PR description or (even better) in the CONTRIBUTING.md file.

File: Jenkinsfile (hunk 1 above)
Comment (YaphetKG) [reply Jenkinsfile:3]: Makes sense , i will try to get with Chuck or Tyler , to make more sense of it (https://drive.google.com/file/d/1HVt74gK4d-JJqo8d7sKmnrRba7viftvY/view)(@43:00) they had given the Architecture group a demo .

File: src/dug/core/parsers/bacpac_parser.py (hunk 2 above)
Comment (YaphetKG) [reply src/dug/core/parsers/bacpac_parser.py:8]: Hmm, that's a good point. We have a bunch of logging setups all over the place, currently all use dug logger. But we may need to do some overhaul on logging in the next round maybe as we refactor some of the NER integration aswell. For now i think changing this one would make it a special maybe (?)

Comment (gaurav) [reply src/dug/core/parsers/bacpac_parser.py:8]: Yeah, that makes sense! It's probably worth opening an issue for this and dealing with all of these there.

**Pull request merged**